packages/disclaimer/src/lib/Disclaimer.tsx

```diff
@@ -0,0 +1,81 @@
+// For bundling purposes, always use the direct import for an mui component, i.e. '@mui/material/xxx'
+import { Typography, TypographyProps } from '@availity/mui-typography';
+import { Link, LinkProps } from '@availity/mui-link';
+import { styled } from '@mui/material/styles';
+
+type AvDisclaimerProps = {
+  accent?: boolean;
+  condensed?: boolean;
```

Comment:
`dense` would be more consistent with Material

Reply:
good call, fixed